runelite-client/src/main/java/net/runelite/client/plugins/bank/BankCalculation.java

```diff
@@ -177,6 +177,12 @@ void calculate()
 					continue;
 				}
 
+				//Limit quantity to 1 because master scroll books share teleport scrolls
+				if (itemId == ItemID.MASTER_SCROLL_BOOK)
```

Comment:
This is still incorrect because it needs to count master scroll book x times, but teleports inside only once